test/jetstream_test.go

```diff
@@ -83,7 +83,7 @@ func RunBasicJetStreamServer() *server.Server {
 	opts := DefaultTestOptions
 	opts.Port = -1
 	opts.JetStream = true
-	opts.NoLog = true
+	opts.NoLog = false
```

Comment:
Do you want the test to print log statements? This should be set to true.

Reply:
Also see my comment about testing debug/trace statements with the use of a dummy logger. Note, we may not have a DummyLogger in the test/ package, but that could be easily copied from server tests.

Reply:
restored as it was before, i think this is some line of code i put in on some other day while debugging and just didnt notice :) gone now.